.github/workflows/gui.yml

```diff
@@ -512,6 +512,7 @@ jobs:
           ENSO_TEST_USER: ${{ secrets.ENSO_CLOUD_TEST_ACCOUNT_USERNAME }}
           ENSO_TEST_USER_PASSWORD: ${{ secrets.ENSO_CLOUD_TEST_ACCOUNT_PASSWORD }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        continue-on-error: true
```

Comment:
The PR is called

> Block integration tests

Shouldn't it be _disable_, _disarm_, _ignore results of_?

As right now I am "blocked by integration tests", and I'd like to be _"unblocked"_.

Reply:
Sure, I changed the name.